nixos/default.nix

```diff
@@ -34,6 +34,7 @@ in
 
 {
   inherit (eval) pkgs config options;
+  inherit vmConfig;
```

Comment:
Is this a leftover from debugging/still working? I don't think it is technically required, is it?

Reply:
It's not technically required, so yea I can remove it from this PR. I do find it *tremendously* useful to have in nixos/default.nix though, FWIW.